docs/application/native/api/iot-headed/5.0/group__Ecore__Con__Client__Group.html

```diff
@@ -173,6 +173,7 @@
   </dd>
 </dl>
 <dl class="return"><dt><b>Returns:</b></dt><dd><code>EINA_TRUE</code> if connected, <code>EINA_FALSE</code> otherwise. </dd></dl>
+<dl class="user"><dt><b>Since :</b></dt><dd>3.0f(2.3.1) </dd></dl>
```

Comment:
It means ... this API was added to 3.0 final release, but, actually, it had been added since 2.3.1.
I think.. no need to use 3.0f(2.3.1), instead of 3.0
will be updated soon after discussion with EFL team.

Reply:
http://10.113.245.21/gerrit/#/c/249709/